package.json

```diff
@@ -1,8 +1,47 @@
 {
+  "name": "web-gphoto2",
+  "version": "1.0.0",
+  "description": "WebAssembly implementation of gphoto2 and libusb to control DSLR cameras over USB on the Web",
+  "type": "module",
+  "main": "src/camera.js",
+  "types": "src/libapi.mjs.d.ts",
```

Comment:
Ah I suspect it might be because of this field that specifies types for the whole package. Try removing it?

Reply:
Hmm it seems that NPM packages pretty much require the 'types' field. If I remove it I get the following error when importing the package:

```
Could not find a declaration file for module 'web-gphoto2-npm-test'. '/home/tjbakker/Documents/dev/vscode/webdev/EditExpress/node_modules/web-gphoto2-npm-test/src/camera.js' implicitly has an 'any' type.
  If the 'web-gphoto2-npm-test' package actually exposes this module, try adding a new declaration (.d.ts) file containing `declare module 'web-gphoto2-npm-test';`ts(7016)
 ```

Reply:
Ah you probably need to follow this instead. It will generate `.d.ts` for npm package from the JS file: https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html

Reply:
I tried throwing a cheeky 
```ts/**
 * @typedef {import('./camera.js').Camera} Camera
 */```
 
Into the libapi.mjs.d.ts file but that didn't work. I could construct a `types.d.ts` file containing the previous class definition, and then also import the type definitions from `libapi.d.ts` so that we have a single file with all type definitions? I'm pretty sure that would work but I'm not sure if that is the normal way to do things.

Reply:
Yeah I guess just creating a separate `types.d.ts` is fine for now. You should be able to use `import('./libapi.mjs')` and such there.

Reply:
Fixed it